content/glossary/sql/between.md

```diff
@@ -0,0 +1,37 @@
+<!--
```

Comment:
remove "sql" from tags and subjects, title from markdown

Reply:
👍🏼